inc/class-wp-cli-command.php

```diff
@@ -99,6 +99,39 @@ public function save( $args, $args_assoc ) {
 			if ( ! empty( $args_assoc['verbose'] ) ) {
 				WP_CLI::line( 'Saving ' . $url );
 			}
+
+			$options = [
+				'user_id' => get_current_user_id(),
+				'context' => '',
+				'action'  => '',
+			];
+
+			$post_id = url_to_postid( $url );
+			if ( ! empty( $post_id ) ) {
+				$post = get_post( $post_id );
+				if ( $post->post_type === 'page' ) {
+					$options['context'] = 'page';
+					$options['action']  = 'wp_cli_netstorage_publish';
```

Comment:
The `$options['action']` value set here, and the value set in the conditional below is the same, so I assume that `wp_cli_netstorage_publish` could just the the default.

Reply:
`$options['actiion']` is only set if
1. Url belong to a post ID or
2. Url is used as path in `netstorage-file` CPT

Otherwise `$options['actiion']` should be empty